src/components/SchemaRow/TopLevelSchemaRow.tsx

```diff
@@ -101,6 +103,12 @@ export const TopLevelSchemaRow = ({
           array of:
         </Box>
 
+        {!isEmpty(validations) && (
+          <Box fontSize="sm" mb={1} mt={-2}>
+            <Validations validations={isRegularNode(schemaNode) ? getValidationsFromSchema(schemaNode) : {}} />
```

Comment:
So if the ternary evaluates to false and the result is `{}` then is it just going to show a little extra whitespace?

Reply:
Works for me!